golang_versions

```diff
@@ -0,0 +1,5 @@
+# This file contains the versions of managed golang releases. 
+# It is used by Lunar tooling to decide which versions of golang binaries should be installed and maintained
+
+# The scheme is <name>::go.lunarway.com/<name>@v0.5.0 as an example
+lunarctl::go.lunarway.com/lunarctl@v0.5.0
```

Comment:
What about renovate? Will it pick this line up as is or do we need to update the config file?

Reply:
Also, be aware that this repo is public

Reply:
I've scoped out renovate for now. I'll add it in a further pr when I've got the scheme right. 

Yep, I know it is public, we do leak the name of the tool and version, but as there are no access I guess it is a minimal risk of exposure. We only really tell that we've got a certain tool available, on a given version, but not what it is.